src/wallet/wallet.h

```diff
@@ -700,7 +700,16 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
 
     bool fFileBacked;
 
-    std::set<int64_t> setKeyPool;
```

Comment:
Fair point. The question is should we do it in this PR or later. But since we significant change the  set's content, It should be changed now (which I will do soon).